Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41206&attempt_id=41390cbc-90a4-416c-bcc5-4147e7b9036b)

Attempted: mysql55, mysql57

<details><summary>Partial log (click to expand)</summary><p>

```
  case DATA_VARCHAR:
  ^~~~
[ 69%] Linking CXX static library libinnobase_embedded.a
[ 69%] Built target innobase_embedded
[ 69%] Linking CXX static library libinnobase.a
[ 69%] Built target innobase
make[1]: *** [CMakeFiles/Makefile2:4664: libmysqld/CMakeFiles/sql_embedded.dir/all] Error 2
make: *** [Makefile:163: all] Error 2
builder for '/nix/store/2qrw4hbs8790g0m0vnvvybf49c7sjg4i-mysql-5.5.60.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/2qrw4hbs8790g0m0vnvvybf49c7sjg4i-mysql-5.5.60.drv' failed
```
</p></details>

--- a/pkgs/development/libraries/qt-5/modules/qtlocation.nix
+++ b/pkgs/development/libraries/qt-5/modules/qtlocation.nix
@@ -4,8 +4,6 @@ qtModule {
   name = "qtlocation";
   qtInputs = [ qtbase qtmultimedia ];
   outputs = [ "bin" "out" "dev" ];
-  # Linking with -lclipper fails with parallel build enabled
-  enableParallelBuilding = false;
   qmakeFlags = stdenv.lib.optional stdenv.isDarwin [
      # boost uses std::auto_ptr which has been disabled in clang with libcxx
      # This flag re-enables this feature

--- a/pkgs/servers/sql/mysql/5.5.x.nix
+++ b/pkgs/servers/sql/mysql/5.5.x.nix
@@ -6,11 +6,11 @@
 let
 self = stdenv.mkDerivation rec {
   name = "mysql-${version}";
-  version = "5.5.58";
+  version = "5.5.60";
 
   src = fetchurl {
     url = "mirror://mysql/MySQL-5.5/${name}.tar.gz";
-    sha256 = "1f890376ld1qapl038sjh2ialdizys3sj96vfn4mqmb1ybx14scv";
+    sha256 = "071xaamqkbscybqzm79gf2w3bkr9lqlzwafis3gzc8w8fkhi4hd3";
   };
 
   patches = if stdenv.isCygwin then [
@@ -47,6 +47,8 @@ self = stdenv.mkDerivation rec {
     "-DINSTALL_MYSQLSHAREDIR=share/mysql"
     "-DINSTALL_DOCDIR=share/mysql/docs"
     "-DINSTALL_SHAREDIR=share/mysql"
+    "-DINSTALL_MYSQLTESTDIR="
+    "-DINSTALL_SQLBENCHDIR="
   ];
 
   NIX_CFLAGS_COMPILE = [ "-fpermissive" ]; # since gcc-7
@@ -57,8 +59,7 @@ self = stdenv.mkDerivation rec {
   '';
   postInstall = ''
     sed -i -e "s|basedir=\"\"|basedir=\"$out\"|" $out/bin/mysql_install_db
-    rm -r $out/mysql-test $out/sql-bench $out/data "$out"/lib/*.a
-    rm $out/share/man/man1/mysql-test-run.pl.1
+    rm -r $out/data "$out"/lib/*.a
   '';
 
   passthru = {

--- a/pkgs/servers/sql/mysql/5.7.x.nix
+++ b/pkgs/servers/sql/mysql/5.7.x.nix
@@ -6,11 +6,11 @@
 let
 self = stdenv.mkDerivation rec {
   name = "mysql-${version}";
-  version = "5.7.20";
+  version = "5.7.22";
 
   src = fetchurl {
     url = "mirror://mysql/MySQL-5.7/${name}.tar.gz";
-    sha256 = "11v4g3igigv3zvknv67qml8in6fjrbs2vnr3q6bg6f62nydm95sk";
+    sha256 = "1wng15j5caz6fsv28avlcxjgq3c5n90ifk79xa0h7jws19dl1f2f";
   };
 
   preConfigure = stdenv.lib.optional stdenv.isDarwin ''
@@ -26,14 +26,14 @@ self = stdenv.mkDerivation rec {
   outputs = [ "out" "static" ];
 
   cmakeFlags = [
+    "-DCMAKE_SKIP_BUILD_RPATH=OFF" # To run libmysql/libmysql_api_test during build.
     "-DWITH_SSL=yes"
     "-DWITH_EMBEDDED_SERVER=yes"
-    "-DWITH_UNITTEST=no"
+    "-DWITH_UNIT_TESTS=no"
     "-DWITH_ZLIB=yes"
     "-DWITH_ARCHIVE_STORAGE_ENGINE=yes"
     "-DWITH_BLACKHOLE_STORAGE_ENGINE=yes"
     "-DWITH_FEDERATED_STORAGE_ENGINE=yes"
-    "-DCMAKE_VERBOSE_MAKEFILE=yes"
     "-DHAVE_IPV6=yes"
     "-DMYSQL_UNIX_ADDR=/run/mysqld/mysqld.sock"
     "-DMYSQL_DATADIR=/var/lib/mysql"
@@ -45,6 +45,7 @@ self = stdenv.mkDerivation rec {
     "-DINSTALL_DOCREADMEDIR=share/mysql"
     "-DINSTALL_SUPPORTFILESDIR=share/mysql"
     "-DINSTALL_MYSQLSHAREDIR=share/mysql"
+    "-DINSTALL_MYSQLTESTDIR="
     "-DINSTALL_DOCDIR=share/mysql/docs"
     "-DINSTALL_SHAREDIR=share/mysql"
   ];
@@ -53,13 +54,10 @@ self = stdenv.mkDerivation rec {
   NIX_LDFLAGS = stdenv.lib.optionalString stdenv.isLinux "-lgcc_s";
 
   prePatch = ''
-    sed -i -e "s|/usr/bin/libtool|libtool|" cmake/libutils.cmake
+    sed -i -e "s|/usr/bin/libtool|libtool|" cmake/merge_archives.cmake.in
   '';
   postInstall = ''
-    sed -i -e "s|basedir=\"\"|basedir=\"$out\"|" $out/bin/mysql_install_db
-    install -vD $out/lib/*.a -t $static/lib
-    rm -r $out/mysql-test
-    rm $out/share/man/man1/mysql-test-run.pl.1 $out/lib/*.a
+    moveToOutput "lib/*.a" $static
     ln -s libmysqlclient${stdenv.hostPlatform.extensions.sharedLibrary} $out/lib/libmysqlclient_r${stdenv.hostPlatform.extensions.sharedLibrary}
   '';
 

